cmd/exposure/main.go

```diff
@@ -48,13 +48,25 @@ func realMain(ctx context.Context) error {
 	}
 	defer env.Close(ctx)
 
-	handler, err := publish.NewHandler(ctx, &config, env)
-	if err != nil {
-		return fmt.Errorf("publish.NewHandler: %w", err)
+	if !(config.EnableV1API || config.EnableV1Alpha1API) {
+		return fmt.Errorf("no APIs enabled, must set one or more of ENABLE_V1_API, ENABLE_V1ALPHA1_API")
 	}
 
 	mux := http.NewServeMux()
-	mux.Handle("/", handler)
+	if config.EnableV1Alpha1API {
```

Comment:
I don't know why I made v1 configurable.

Changed this so that v1 is always on, but you can disable v1alpha1 in advance of us removing it from the codebase.

Reply:
👍